### DISCUSSION TITLE
No AP showinig up after the installation

### DISCUSSION THREAD

**Discussion by Laxx4** [RaspAP/raspap-webgui#984] [Q&A]
Hi 
OS: Ubuntu 20.04 LTS

when i installed RaspAP, I can't see any Wifi network named "raspi-webgui" ,even after a reboot.
(i have installed it like shown here: https://docs.raspap.com/manual/ under manual installation where it sais for Ubuntu)

Why i don't just use the built in hotspot feature in Ubuntu?
Well, the option isn't showing up there (it's just like this:
![image](https://user-images.githubusercontent.com/68148159/128742548-f955d59b-f21c-4166-83f2-c74fa1d5f8e9.png)
 , i checked with `iw list` if my wifi card does support AP mode, and it does... (see picture)
![Bild_2021-08-09_183916](https://user-images.githubusercontent.com/68148159/128742063-4c1943d4-338c-485d-88ad-4239a9e7a266.png)
Any idea about what's my problem? And if so, any to solve it? 

Thanks in advance

**Comment by billz** [comment]
Start by troubleshooting https://docs.raspap.com/ap-basics/#troubleshooting
